src/utils/logging.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export interface LogInterface {
+    debug(msg: string, ...supportingData: any[]): void;
+    warn(msg: string, ...supportingData: any[]): void;
+    error(msg: string, ...supportingData: any[]): void;
+    info(msg: string, ...supportingData: any[]): void;
+}
+
+export enum LogLevel {
+    debug,
```

Comment:
Enum values in TypeScript are [typically written PascalCase](https://www.typescriptlang.org/docs/handbook/enums.html), e.g. `Debug`, `Info`. Let's stick with that

Reply:
Done